src/D2L.Bmx/Authenticate.cs

```diff
@@ -124,19 +126,38 @@ IOktaApi oktaApi
 		var currTime = DateTimeOffset.Now;
 		return sourceCache.Where( session => session.ExpiresAt > currTime ).ToList();
 	}
-
-	private static int PromptMfa( MfaOption[] mfaOptions ) {
-
+	private static int MatchDefaultMFAoption( MfaOption[] mfaOptions, string DefautMFAProvider, string DefaultMFAmethod ) {
+		return Array.IndexOf( mfaOptions,
+			Array.Find( mfaOptions, option =>
+				option.Provider == DefautMFAProvider && option.Name == DefaultMFAmethod
+)
+		);
+	}
+	private static int PromptMfa( MfaOption[] mfaOptions, string DefautMFAProvider, string DefaultMFAmethod ) {
```

Comment:
```suggestion
	private static int PromptMfa( MfaOption[] mfaOptions, string defaultMfaProvider, string defaultMfaMethod ) {
```
Looks like the other areas of code have `Mfa` instead of `MFA` all caps

Reply:
Also want to use camelCase

Reply:
Sure will change that